src/backend/distributed/sql/citus--10.0-3--10.1-1.sql

```diff
@@ -24,3 +24,21 @@ ALTER TABLE pg_catalog.pg_dist_rebalance_strategy ADD COLUMN improvement_thresho
 UPDATE pg_catalog.pg_dist_rebalance_strategy SET improvement_threshold = 0.5 WHERE name = 'by_disk_size';
 
 #include "udfs/get_rebalance_progress/10.1-1.sql"
+
+-- use streaming replication when replication factor = 1
+WITH replicated_shards AS (
+    SELECT shardid
+    FROM pg_dist_placement
+    WHERE shardstate = 1
+    GROUP BY shardid
+    HAVING count(*) <> 1 ),
+replicated_relations AS (
+    SELECT DISTINCT logicalrelid
+    FROM pg_dist_shard
+    JOIN replicated_shards
+    USING (shardid)
+)
+UPDATE pg_dist_partition
+SET repmodel = 's'
+WHERE logicalrelid NOT IN (SELECT * FROM replicated_relations)
```

Comment:
wouldn't this include reference tables?

Reply:
Let's include `WHERE repmodel = 'c'`

Reply:
The update query skips over reference tables.

`SELECT * FROM replicated_relations` will include reference tables, but they will get filtered-out on the filter `partmethod = 'h'` several lines below.